src/web/make-graph.rkt

```diff
@@ -153,16 +158,18 @@
           ;[style "rotate: 270deg"]
           ) "?"))
        (ol ([class "history"])
-        ,@(render-history end-alt (mk-pcontext newpoints newexacts)
-                          (mk-pcontext points exacts) (test-context test))))
+        ,@(render-history end-alt train-pctx test-pctx ctx)))
 
-      ,(if (not (null? other-alts))
+      ,(if (not (null? (cdr end-alts)))
```

Comment:
I mean, it's fine, but definitely this would look saner as `(if (> (length end-alts) 1) ...)` even if hypothetically that's slower.